doc/source/examples/dem/granular-dam-break/granular-dam-break.rst

```diff
@@ -0,0 +1,218 @@
+
+==================================
+Granular dam-break
+==================================
+
+This example simulates the dam-break flow of a dry granular material using the discrete element method (DEM). More information regarding the DEM parameters are given in the Lethe documentation, i.e. `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Three-dimensional problem
+- Moving floating mesh
+- Load-balancing
+
+
+
+Files used in this example
+----------------------------
+
+- Parameters file for the problem (H=20cm): ``examples/dem/3d-dam-break/granular-dam-break.prm``
+- Parameters file for the H=40cm bed: ``examples/dem/3d-dam-break/granular-dam-break.prm``
+- GMSH geo file: ``examples/dem/3d-dam-break/square.geo``
+- GMSH mesh file: ``examples/dem/3d-dam-break/square.msh``
+
+
+
+
+Description of the case
+-----------------------
+
+This example simulates the dam-break flow of a dry powder. First, a packing of particles is generated by inserting particles behind a sluice gate. After 0.5s, the sluice gate rises and the particles are able to flow freely, producing a dam break flow. We compare the results we obtain with those of `Evangelista et al. <https://www.cabdirect.org/cabdirect/abstract/20153319263>`_. To ensure that the simulation remains computationnaly efficient, we will scale-up the sand particles used by Evangelista et al. from a diameter of 1.6mm to 10mm. This reduces their number by a factor of approximately 250x.We first simulate case B1 of Evangelista et al., with a bed length of 60cm and bed height of 20cm. As an extension, we also simulate a bed height of 40cm (case B3).
+
+
+
+Parameter file
+--------------
+
+Mesh
+~~~~~
+
+The domain we simulate is a rectangular box which is :math:`2\times0.4\times0.4` meters which is made using the deal.ii grid generator.  The grid is refined 4 times using the ``set initial refinement`` parameters.
+
+.. code-block:: text
+
+  subsection mesh
+    set type               = dealii
+    set grid type          = subdivided_hyper_rectangle
+    set grid arguments     = 5,1,1 : 0,0,0 : 2,0.4,0.4 : false
+    set initial refinement = 4
+  end
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles are mono-dispersed with a radius of 0.005 m and a density of 2560 :math:`kg/m3`, respectively. We insert 60k particles to ensure an initial height of 0.2m in order to match the B1 experiment of Evangelista et al. The friction coefficient and the rolling coefficient are the most important parameters in this simulation, since they control the final repose angle of the particles as well as the speed of the progression of the dam-break flow. Since sand is generally non-spherical, we choose relatively high values for the friction and the rolling friction coefficients.
```

Comment:
```suggestion
The particles are mono dispersed with a radius of 0.005 m and a density of 2560 :math:`kg/m^3`, respectively. We insert 60k particles to ensure an initial height of 0.2m in order to match the B1 experiment of Evangelista et al. The friction coefficient and the rolling coefficient are the most important parameters in this simulation, since they control the final repose angle of the particles as well as the speed of the progression of the dam-break flow. Since sand is generally non-spherical, we choose relatively high values for the friction and the rolling friction coefficients.
```